pkgs/development/python-modules/cooldict/default.nix

```
@@ -0,0 +1,23 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "cooldict";
+  version = "unstable-2019-10-29";
+
+  src = fetchFromGitHub {
+    owner = "zardus";
+    repo = pname;
+    rev = "32d56888b555f4513a0e41193bfa49e0e5497a7b";
+    sha256 = "1nwf2gd6mx9nm0s9bg0xd5a7b286ph0789fnadri5yd8qqfwr50i";
+  };
+
```

I would be honest about the amount of testing the package has, and then add pythonImportsCheck to make sure that it doesn't immediate error when someone tries to use it.

```suggestion

  # no tests in repo
  doCheck = false;
  
  # verify imports still work
  pythonImportsCheck = [ "cooldict" ];
```